static/css/tips_board.css

```diff
@@ -0,0 +1,36 @@
+    <style type="text/css">
+      .author,
+      .eom {
+        color: grey;
+      }
+      .grid{
+          height:100%;
+          width:100%;
+      }
+      .item {
+        width: 50%;
+        background-color: grey;
+        position:relative;
+      }
+      .item p {
+        color: white;
+        text-align: center;
+      }
+      .item * {
+        color: white;
+        text-align: center;
+      }
+      .item--width2 {
+        width: 50%;
+      }
+      .date{
+          position: absolute;
+          top:0;
+          right:0;
+      }
+      .edit{
+        position: absolute;
+        bottom:0;
+        left:0;
+      }
+    </style>
```

Comment:
you need to add a new line at the end of the file

Reply:
fixed